src/main/java/com/rockset/client/RocksetClient.java

```diff
@@ -45,7 +51,7 @@ public RocksetClient(String apiKey, String apiServer, String application) {
         new ApiClient()
             .setApiKey(apiKey)
             .setApiServer(apiServer)
-            .setVersion("0.10.3") // TODO: figure out how we can set this dynamically.
+            .setVersion(implementationVersion)
```

Comment:
I'm guessing the implementation version will be changed after a new version jar is published?

Reply:
Yes, it is defined in the pom file